doc/reactive-migration.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
@ajpinedam can you move reactive-migration.md into the correct folder please
https://github.com/unoplatform/uno.extensions/tree/main/doc/Learn/Mvux

Reply:
done :)